operators/pkg/about/info.go

```diff
@@ -21,12 +21,14 @@ const (
 	UUIDCfgMapKey = "uuid"
 )
 
+var defaultOperatorNamespaces = []string{"elastic-namespace", "elastic-namespace-operators"}
```

Comment:
Are we not also using `elastic-system`? I am a bit concerned that we are enumerating these namespaces in the code as they might change quite a bit and it feels somehow wrong. I wonder what alternatives we have?

Reply:
Well seen. Yes, it's `elastic-system` instead of `elastic-namespace`.

I share a bit your concern but I have no other idea for the moment.
We do not yet know exactly what the story for the operator namespaces is.
I'm confident we will not forget this when we will know more.